templates/nextflow-aurora-mysql.yaml

```diff
@@ -87,12 +86,12 @@ Resources:
           - - !Sub 'arn:aws:iam::${AWS::AccountId}:root'
             - !ImportValue
               'Fn::Sub': '${AWS::Region}-workflows-nextflow-ci-service-account-ServiceRoleArn'
-            - !FindInMap [Accounts, !Ref AWS::AccountId, AdminRole]
+            - !Join [",", !Ref AccountAdminArns]
```

Comment:
you can probably remove `'${AWS::Region}-workflows-nextflow-ci-service-account-ServiceRoleArn'`  since it looks like it's already being passed into `AccountAdminArns`

Reply:
Good catch! My eyes went straight for `!FindInMap`.